presto-docs/src/main/sphinx/connector/kudu.rst

```diff
@@ -19,7 +19,7 @@ Compatibility
 Connector is compatible with all Apache Kudu versions starting from 1.0.
```

Comment:
Please fix the commit message, it wraps in GitHub.

Separate commit to two. One with Kudu client update and the other with properties.